README.md

```diff
@@ -23,6 +24,46 @@
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
+## Supported Apache Spark
+
+<table>
```

Comment:
This section can grow arbitrarily large.  Should we surround the table with something like the following? 

<details open="open"><summary>Supported Versions</summary>
<p>
<table>
    <thead>
        <tr>
            <th>Apache Spark</th>
            <th>.NET for Apache Spark</th>
        </tr>
    </thead>
    <tbody align="center">
        <tr>
            <td >2.3.0</td>
            <td rowspan=6><a href="https://github.com/dotnet/spark/releases/tag/v0.2.0">v0.2.0</a></td>
        </tr>
        <tr>
            <td>2.3.1</td>
        </tr>
        <tr>
            <td>2.3.2</td>
        </tr>
        <tr>
            <td>2.3.3</td>
        </tr>
        <tr>
            <td>2.4.0</td>
        </tr>
        <tr>
            <td>2.4.1</td>
        </tr>
        <tr>
            <td>2.4.2</td>
            <td><a href="https://github.com/dotnet/spark/issues/60">Not supported</a></td>
        </tr>
        <tr>
            <td>2.4.3</td>
            <td>master branch</td>
        </tr>
    </tbody>
</table></p>
</details>

Reply:
Yea, that's a good option. (Maybe I should change it to "Click to see all the supported versions")?

Do I also need to collapse like this or not?:

<table>
    <thead>
        <tr>
            <th>Apache Spark</th>
            <th>.NET for Apache Spark</th>
        </tr>
    </thead>
    <tbody align="center">
        <tr>
            <td >2.3.*</td>
            <td rowspan=3><a href="https://github.com/dotnet/spark/releases/tag/v0.2.0">v0.2.0</a></td>
        </tr>
        <tr>
            <td>2.4.0</td>
        </tr>
        <tr>
            <td>2.4.1</td>
        </tr>
        <tr>
            <td>2.4.2</td>
            <td><a href="https://github.com/dotnet/spark/issues/60">Not supported</a></td>
        </tr>
        <tr>
            <td>2.4.3</td>
            <td>master branch</td>
        </tr>
    </tbody>
</table>

Reply:
That table looks better and more compact.  Let's surround it with the <details> with the default set to "open" for now and we can the default behavior to closed once it gets large.

Reply:
If you collapse it already, why hide it? I wonder if it's good to be explicit. I don't really know how many people will explicitly click to open it. 

The other option is to pivot it the other way i.e., 
v0.2.0 -> {versions supported}
Not supported -> {versions not supported}
master branch -> {current version}

Reply:
I will collapse without hiding and we can revisit if the table becomes too big.